ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -239,237 +238,234 @@ This program is distributed under the Artistic License.
-    $globals->{Css}       =          $opts->{css}        if defined $opts->{css};
-    $globals->{Header}    =          $opts->{header}     if defined $opts->{header};
-    $globals->{Htmldir}   =  unixify($opts->{htmldir})   if defined $opts->{htmldir};
-    $globals->{Htmlroot}  =  unixify($opts->{htmlroot})  if defined $opts->{htmlroot};
-    $globals->{Doindex}   =          $opts->{index}      if defined $opts->{index};
-    $globals->{Podfile}   =  unixify($opts->{infile})    if defined $opts->{infile};
-    $globals->{Htmlfile}  =  unixify($opts->{outfile})   if defined $opts->{outfile};
-    $globals->{Poderrors} =          $opts->{poderrors}  if defined $opts->{poderrors};
-    $globals->{Podroot}   =  unixify($opts->{podroot})   if defined $opts->{podroot};
-    $globals->{Quiet}     =          $opts->{quiet}      if defined $opts->{quiet};
-    $globals->{Recurse}   =          $opts->{recurse}    if defined $opts->{recurse};
-    $globals->{Title}     =          $opts->{title}      if defined $opts->{title};
-    $globals->{Verbose}   =          $opts->{verbose}    if defined $opts->{verbose};
+    $self->{Backlink}  =          $opts->{backlink}   if defined $opts->{backlink};
+    $self->{Cachedir}  =  unixify($opts->{cachedir})  if defined $opts->{cachedir};
+    $self->{Css}       =          $opts->{css}        if defined $opts->{css};
+    $self->{Header}    =          $opts->{header}     if defined $opts->{header};
+    $self->{Htmldir}   =  unixify($opts->{htmldir})   if defined $opts->{htmldir};
+    $self->{Htmlroot}  =  unixify($opts->{htmlroot})  if defined $opts->{htmlroot};
+    $self->{Doindex}   =          $opts->{index}      if defined $opts->{index};
+    $self->{Podfile}   =  unixify($opts->{infile})    if defined $opts->{infile};
+    $self->{Htmlfile}  =  unixify($opts->{outfile})   if defined $opts->{outfile};
+    $self->{Poderrors} =          $opts->{poderrors}  if defined $opts->{poderrors};
+    $self->{Podroot}   =  unixify($opts->{podroot})   if defined $opts->{podroot};
+    $self->{Quiet}     =          $opts->{quiet}      if defined $opts->{quiet};
+    $self->{Recurse}   =          $opts->{recurse}    if defined $opts->{recurse};
+    $self->{Title}     =          $opts->{title}      if defined $opts->{title};
+    $self->{Verbose}   =          $opts->{verbose}    if defined $opts->{verbose};
 
     warn "Flushing directory caches\n"
         if $opts->{verbose} && defined $opts->{flush};
-    $globals->{Dircache} = "$globals->{Cachedir}/pod2htmd.tmp";
+    $self->{Dircache} = "$self->{Cachedir}/pod2htmd.tmp";
     if (defined $opts->{flush}) {
-        1 while unlink($globals->{Dircache});
+        1 while unlink($self->{Dircache});
     }
-    return $globals;
+    return $self;
 }
 
 sub refine_globals {
-    my $globals = shift;
-    require Data::Dumper if $globals->{verbose};
+    my $self = shift;
+    require Data::Dumper if $self->{verbose};
```

Comment:
Data::Dumper required here.

Reply:
Accepted; handled in 1c6c07161d40fc4b5a3762c4a71cbfc8e62e881d